src/components/Post/Details.tsx

```diff
@@ -19,6 +21,18 @@ export const postDetailsTestIds = {
 };
 
 const PostDetails = ({ post }: PostDetailsProps) => {
+  const renderCommaSeparatedTags = () => {
+    const tags = post.tags.map((tag) => (
+      <Link key={tag} href={`/tags/${tag}`}>
+        <a className="hover:underline" href={`/tags/${tag}`}>
+          {tag}
+        </a>
+      </Link>
+    ));
+
+    return intersperse(tags, ', ');
```

Comment:
Should we turn them into a `ul` list of [badges](https://daisyui.com/components/badge/) instead?
We can create a `TagList` component for this also.
And I think we can add this section to the Card of the TIL on the home page as well 🤔

Reply:
sure, done in af489b31147a1a01736b42f0e64a223f449f368c, Thank you!